platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -255,6 +257,11 @@ public static PlatformSchedulers create(
                         .withUnhandledTaskCapacity(config.issDetectorUnhandledCapacity())
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
+                        .cast(),
+                model.schedulerBuilder("appCommunicationComponent")
+                        .withType(TaskSchedulerType.SEQUENTIAL_THREAD)
```

Comment:
This scheduler needs to have an unhandled task capacity set, otherwise the queue is unbounded

Reply:
and the capacity should be the same as it was

Reply:
set it to the previous default of 1000.